.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Publish: Publish to CDN'
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+  # This workflow is intended to be called from the primary publish workflow
+  # automatically. The manual trigger is maintained for partially failed workflows
+  # or edge cases that require this be run independently.
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+env:
+  GH_ACTION_ROLE: arn:aws:iam::073865013699:role/CDN_Autocomplete_UI_GHA_Role
+  AWS_REGION : us-east-1
+
+jobs:
+  publish-to-cdn:
+    name: Publish to CDN
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.x
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build compiled package for publishing
+        run: npm run compile
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-session-name: cdn-publish-action
+          role-to-assume: ${{ env.GH_ACTION_ROLE }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Copy files to CDN
+        run: |
+          aws s3 sync ./lib/cjs s3://constructor-frontend-dev/ui/autocomplete --cache-control 'public, max-age=1800' --acl 'public-read'
```

Comment:
**Issue**: I think we're uploading the wrong file. 

We need to upload the `dist/constructorio-ui-autocomplete-bundled.js` instead. That's the only file needed for bundled usage.

Reply:
Oh, great catch! I also renamed the file to upload it to the proper path (e.g. `/v1.0.0.js`)

845044097dd3c0983f73eccf1a47b12bf1196a1f